README.md

```diff
@@ -311,9 +272,33 @@ Attributes Kit always takes the same parameters.
 
 **Congratulations!** You are all set to start developing now! :rocket:
 
-## Playground
+---
+
+# Compatibility
 
-## Testing
+| Package                 | Version |
+|:------------------------|:--------|
+| React | `0.14+` |
+| React DOM | `0.14+` |
+| Node | `0.10.40+` |
```

Comment:
I think it will work with any of `0.10.x`, but yes!
Also, perhaps we should write that node is required only for development purposes. Any npm-compatible version will work with it to use the distribution.